Makefile

```
@@ -52,9 +58,10 @@ pkg/rest/zz_docs: $(wildcard pkg/rest/*.go pkg/korrel8r/*.go) $(SWAG)
 	$(SWAG) fmt pkg/rest
 	@touch $@
 
-lint: $(VERSION_TXT) $(GOLANGCI_LINT) $(SHFMT) ## Run the linter to find and fix code style problems.
+lint: $(VERSION_TXT) $(GOLANGCI_LINT) $(SHFMT) shellcheck ## Run the linter to find and fix code style problems.
```

Use Make dependenciees to download shellcheck:
```suggestion
SHELLCHECK=$(LOCALBIN)/shellcheck
lint: $(VERSION_TXT) $(GOLANGCI_LINT) $(SHFMT) $(SHELLCHECK) ## Run the linter to find and fix code style problems.
```

$(SHELLCHECK):
        hack/download_shellcheck.sh $(LOCALBIN)/shellcheck